CI failed randomly on `Build_image`

**Description**
The CI failed during its automatic launch.

**Expected behavior**
Ci should pass every time

**Current behavior**
An error is raised [here an example on bump merge](https://github.com/meilisearch/meilisearch-digitalocean/runs/3130618318?check_suite_focus=true)


**Screenshots or Logs**
```
Run python3 tools/build_image.py $IMAGE_NAME
  python3 tools/build_image.py $IMAGE_NAME
  shell: /bin/bash -e {0}
  env:
    DIGITALOCEAN_ACCESS_TOKEN: 
    REQUEST_TIMEOUT_ENV_VAR: 30
    pythonLocation: /Users/runner/hostedtoolcache/Python/3.7.11/x64
    IMAGE_NAME: meilisearch_digitalocean_ci_test_22-07-2021-04-16-02
Creating droplet...
Traceback (most recent call last):
  File "tools/build_image.py", line 20, in <module>
    droplet.create()
  File "/Users/runner/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/digitalocean/Droplet.py", line 568, in create
    data = self.get_data("droplets/", type=POST, params=data)
  File "/Users/runner/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/digitalocean/Droplet.py", line 165, in get_data
    data = super(Droplet, self).get_data(*args, **kwargs)
  File "/Users/runner/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/digitalocean/baseapi.py", line 216, in get_data
    req = self.__perform_request(url, type, params)
  File "/Users/runner/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages/digitalocean/baseapi.py", line 92, in __perform_request
    raise TokenError("No token provided. Please use a valid token")
digitalocean.TokenError: No token provided. Please use a valid token
Error: Process completed with exit code 1.
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -20,9 +20,9 @@ jobs:
       DIGITALOCEAN_ACCESS_TOKEN: ${{secrets.DIGITALOCEAN_ACCESS_TOKEN}}
       REQUEST_TIMEOUT_ENV_VAR: 30
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Set up Python 3.7
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: 3.7
     - name: Install dependencies
@@ -41,9 +41,9 @@ jobs:
     name: pylint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Set up Python 3.7
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
         with:
           python-version: 3.7
       - name: Install dependencies

